gamebook/game_manager.py

```diff
@@ -0,0 +1,42 @@
+from gamebook.terminal_output import TerminalOutput
+from gamebook.terminal_input import TerminalInput
+import sys
+
+
+class GameManager(object):
+    def __init__(self, scene):
+        self.scene = scene
+        self.output_instance = TerminalOutput()
+        self.input_instance = TerminalInput()
+
+    def start(self, scene_import):
+        while True:
+            next_scene_name = self.run_scene()
+            self.scene = self.get_next_scene(next_scene_name, scene_import)
+
+    def get_next_scene(self, scene_name, scene_import):
+        for scene in scene_import:
+            if scene.get_name() == scene_name:
+                return scene
+
+    def run_scene(self):
+
+        self.output_instance.output(self.scene.show_desc())
+
+        if len(self.scene.options) == 0:
+            sys.exit()
+
+        user_input = self.input_instance.\
+            ask_for_user_inputs(self.scene.options)
+
+        while True:
+            try:
+                next_scene = self.scene.next_scene_draw(user_input)
+                if isinstance(next_scene, str):
+                    return next_scene
+                return next_scene.get_scene_name()
+
+            except ValueError:
```

Comment:
What if someone wants to quit the game? (no need to solve now, just think about it).

Reply:
i can add a feature for input when the user types "quit" it will exit the game

Reply:
add an issue